ads/google/a4a/traffic-experiments.js

```diff
@@ -165,9 +173,24 @@ function maybeSetExperimentFromUrl(win, element, experimentName,
     '0': null, // TODO Ensure does not generate exp id
     '1': controlBranchId,
     '2': treatmentBranchId,
+    '3': hasLaunched(win, element) ? treatmentBranchId : controlBranchId, // DF
   };
   if (argMapping.hasOwnProperty(arg)) {
     forceExperimentBranch(win, experimentName, argMapping[arg]);
+    if (arg == '3') {
```

Comment:
Yeah, I thought in the case for '3' both control AND x_COUNT_ON_RENDER was desired. I didn't get that it was "send just x_COUNT_ON_RENDER" and then that would be made server-side to behave like control.